models/model.topics.js

```diff
@@ -0,0 +1,10 @@
+const db = require('../db/connection');
+
+exports.getAllTopics = () => {
+  return db.query('SELECT * FROM topics;')
+  .then(({ rows }) => {
+    if (rows.length > 0) {
```

Comment:
Nice deconstruction of `rows` 👍 
The if statement isn't needed here because we can always expect there to be some topics